modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -335,7 +341,8 @@ public void addOrUpdateExtensionDynamicObjectDefinitionTableValues(
 
 		_validateValues(
 			user.isGuestUser(), objectDefinition.getObjectDefinitionId(), null,
-			objectDefinition.getClassName(), serviceContext, userId, values);
+			objectDefinition.getClassName(), serviceContext,
+			WorkflowConstants.STATUS_APPROVED, userId, values);
```

Comment:
If I understand, the _primaryKey_ is the _objectEntryId_. In this case, do we need to pass the status to _validateValues method?

Reply:
With the `primaryKey` we can get the previous old status, not the new one